utils/gyb_syntax_support/DeclNodes.py

```diff
@@ -443,7 +443,10 @@
              Child('GenericWhereClause', kind='GenericWhereClause',
                    is_optional=True),
              # the body is not necessary inside a protocol definition
-             Child('Accessor', kind='AccessorBlock', is_optional=True),
+             Child('Accessor', kind='Syntax', is_optional=True,
+                   node_choices=[
+                      Child('Accessors', kind='AccessorBlock'),
+                      Child('Getter', kind='CodeBlock')]),
```

Comment:
This aligns with the language reference:
https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#//appleref/swift/grammar/variable-declaration
> variable-declaration → variable-declaration-head variable-name type-annotation code-block
> variable-declaration → variable-declaration-head variable-name type-annotation getter-setter-block

CC: @nkcsgexi